package.json

```diff
@@ -8,7 +8,7 @@
     "build-prod": "yarn build-prod:quiet --progress",
     "build-photon": "webpack --config res/photon/webpack.config.js",
     "lint": "run-p lint-js lint-css",
-    "lint-js": "eslint *.js bin src --report-unused-disable-directives",
+    "lint-js": "eslint *.js bin src --report-unused-disable-directives --cache --cache-strategy content",
     "lint-fix-js": "yarn lint-js --fix",
     "lint-css": "stylelint \"src/**/*.css\" \"res/**/*.css\"",
```

Comment:
I didn't do it for stylelint because there seems to be some issues, see https://github.com/stylelint/stylelint/issues/4314